test/parallel/test-buffer-indexof.js

```diff
@@ -183,59 +183,59 @@ assert.strictEqual(Buffer.from('aaaa00a').indexOf('3030', 'hex'), 4);
   // test usc2 encoding
   const twoByteString = Buffer.from('\u039a\u0391\u03a3\u03a3\u0395', 'ucs2');
 
-  assert.strictEqual(8, twoByteString.indexOf('\u0395', 4, 'ucs2'));
-  assert.strictEqual(6, twoByteString.indexOf('\u03a3', -4, 'ucs2'));
-  assert.strictEqual(4, twoByteString.indexOf('\u03a3', -6, 'ucs2'));
-  assert.strictEqual(4, twoByteString.indexOf(
-    Buffer.from('\u03a3', 'ucs2'), -6, 'ucs2'));
+  assert.strictEqual(twoByteString.indexOf('\u0395', 4, 'ucs2'), 8);
+  assert.strictEqual(twoByteString.indexOf('\u03a3', -4, 'ucs2'), 6);
+  assert.strictEqual(twoByteString.indexOf('\u03a3', -6, 'ucs2'), 4);
+  assert.strictEqual(twoByteString.indexOf(
+    Buffer.from('\u03a3', 'ucs2'), -6, 'ucs2'), 4);
   assert.strictEqual(-1, twoByteString.indexOf('\u03a3', -2, 'ucs2'));
```

Comment:
The order needs to be swapped for -1 too - on this line and other lines

Reply:
I thought that too, but the code an learn task was to change only a specific set of lines.